search_func/Source/utils.cpp

```diff
@@ -0,0 +1,29 @@
+#include "utils.hpp"
+
+
+int find(std::vector<std::string> vector, std::string value)
+{
+    int i;
+	for(i = 0; i < (int)vector.size(); ++i)
+	{
+		if(vector[i] == value)
+		{
+			break;
+		}
+	}
+	if(i > (int)vector.size()) return -1;
```

Comment:
po co to rzutowanie ?

Reply:
Żeby nie było warningów, a chyba nawet miałam kiedyś errora jak próbowałam porównać size_t z intem, ale nie jestem pewna.

Reply:
> Żeby nie było warningów, a chyba nawet miałam kiedyś errora jak próbowałam porównać size_t z intem, ale nie jestem pewna.

https://afterthoughtsoftware.com/posts/C-sign-compare
Jak porównujesz signeda z unsignedem, zdarzyć się może tak, że da ci to dziwny wynik.
Często zdarza się jak masz taki pseudo iterator `int i` i porównujesz to w pętli z vec.size().
Oczywiście kompilatory domyślnie nawet nie dają o tym żadnych ostrzeżeń.
Lepszą opcją byłoby tu chyba po prostu zastąpienie `int i` czymś unsignedowym